client/src/components/Landing/Landing.js

```diff
@@ -33,9 +34,16 @@ const Landing = () => {
       setIsLoading(true);
       setShowDescription(false);
 
-      // const TRANSLATE_END_POINT = "";
+      const ENDPOINT = `?sl=${formData.sl}&dl=${formData.dl}&code=${formData.code}`;
 
-      history.push("?sl=c&dl=javascript&code=console.log()");
+      const results = await fetch(`http://localhost:5000/api/v1${ENDPOINT}`);
```

Comment:
in the near future I also need to figure out how to refactor the server port so that the server and client apps are retrieving 1 variable value from the same location